apps/api/src/app/view/students/queries/get-students.handlers.ts

```diff
@@ -0,0 +1,13 @@
+import { IQueryHandler, QueryHandler } from '@nestjs/cqrs';
+import { Inject } from '@nestjs/common';
+import { StudentModelService } from '../../../models/students/student.service';
+
+export class GetStudentsQuery { }
+
+@QueryHandler(GetStudentsQuery)
+export class GetStudentsHandler implements IQueryHandler<GetStudentsQuery>{
+    @Inject() private readonly _studentModelService: StudentModelService;
+    async execute() {
+        return this._studentModelService.model.find().exec();
```

Comment:
Replace `.model.find().exec();` to getAll(), create `getAll(){}` function inside StundentModelService and return `    return this._model.find().exec();`